R/sim_gs_n.R

```diff
@@ -112,16 +112,15 @@
 #' )
 #'
 #' # Test 1: regular logrank test
-#' sim_gs_n(
-#'   n_sim = 3,
-#'   sample_size = 400,
-#'   enroll_rate = enroll_rate,
-#'   fail_rate = fail_rate,
-#'   test = wlr,
-#'   cut = list(ia1 = ia1, ia2 = ia2, fa = fa),
-#'   seed = 2024,
-#'   weight = fh(rho = 0, gamma = 0)
-#' )
+# sim_gs_n(
+#   n_sim = 3,
+#   sample_size = 400,
+#   enroll_rate = enroll_rate,
+#   fail_rate = fail_rate,
+#   test = wlr,
+#   cut = list(ia1 = ia1, ia2 = ia2, fa = fa),
+#   weight = fh(rho = 0, gamma = 0)
+# )
```

Comment:
Please add back the apostrophe (`#'`) as this removed this example from the documentation. Also note that if you are using RStudio, you can execute this code with Ctrl-Enter without having to uncomment it.